Tasks/sum.js

```diff
@@ -1,20 +1,13 @@
 // Sum all numbers from an array
+"use strict";
 
-const k = (sum = function (s = () => {}) {
-  sum = [0];
-  let k = 5;
-  for (i of s) {
-    let t = typeof i;
-    if (t === 'number') {
-      if (sum.length > 0) {
-        const new_Sum = sum[sum.length - 1] + i;
-        sum.push(new_Sum);
-      }
-    }
+const sumOfArray = (args) => {
```

Comment:
```suggestion
const sumOfArray = args => {
```
If there is one function parameter, you can pass it without brackets